README.md

```diff
@@ -268,6 +268,36 @@ gcloud dataproc clusters create <CLUSTER NAME> \
   - Replace `<CLUSTER NAME>` with the cluster name you wish to use
   - Replace `<REGION>` with the region you wish to use
 
+
+## Running on AWS EMR
```

Comment:
This is placed in between 2 sections of dataproc (the next heading, `### Debugging problems`, belongs to dataproc)

Reply:
Good catch, fixed